wkcuber/api/layer.py

```diff
@@ -193,7 +339,6 @@ def delete_mag(self, mag: Union[int, str, list, tuple, np.ndarray, Mag]) -> None
             )
 
         del self._mags[mag]
-        self.dataset.properties._delete_mag(self.name, mag.to_layer_name())
```

Comment:
Don't we also need to re-export the datasource-properties.json? Please double-check all places where properties have been changed before, that we don't forget any re-exports.